src/Select.php

```diff
@@ -18,7 +18,7 @@
  * @method array|false fetchAll()
  * @method int fetchAffected()
  * @method array|false fetchColumn(int $column = 0)
- * @method array|false fetchGroup(int $style = PDO::FETCH_COLUMN)
+ * @method array|false fetchGroup(int $style = 7) // 7 = PDO::FETCH_COLUMN
```

Comment:
What is the reason for this change? It seems better the way it was.

Reply:
phpstan somehow could not detect the constant and was throwing errors. That is the only reason for this change.

Reply:
The import may be missing and `Atlas\Query\PDO` may not exist.